### PR TITLE
feat(photo): add new endpoint

### PR DESCRIPTION
**Pull request by libhide**

### CODE REVIEW

**Comment by libhide** [comment]
* **#40** <a href="https://app.graphite.dev/github/pr/libhide/uttam/40?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#39** <a href="https://app.graphite.dev/github/pr/libhide/uttam/39?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#38** <a href="https://app.graphite.dev/github/pr/libhide/uttam/38?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @libhide and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: app/src/main/kotlin/com/ratik/uttam/data/UnsplashApi.kt [hunk 1]
```diff
@@ -26,4 +26,12 @@ interface UnsplashApi {
         @Query("w") w: Int? = null,
         @Query("h") h: Int? = null,
     ): Response<PhotoApiModel>
+
+    @GET("photos/random")
+    suspend fun getRandomPhotoSus2(
+        @Query("client_id") clientId: String?,
```
Comment (BhavyaRattan) [inline app/src/main/kotlin/com/ratik/uttam/data/UnsplashApi.kt:32]: update requested

**Review by BhavyaRattan — COMMENTED** [review]
Updates required

**Review by BhavyaRattan — APPROVED** [review]

**Comment by libhide** [comment]
### Merge activity

* **May 9, 10:59 AM EDT**: @libhide started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/libhide/uttam/38).

**Pull request merged**